src/packages/excalidraw/webpack.prod.config.js

```diff
@@ -89,6 +99,10 @@ module.exports = {
   },
   plugins: [
     ...(process.env.ANALYZER === "true" ? [new BundleAnalyzerPlugin()] : []),
+    new webpack.SourceMapDevToolPlugin({
+      filename: "sourcemaps/[name].js.map",
+      publicPath: "https://unpkg.com/@excalidraw/excalidraw@0.2.2/dist/",
```

Comment:
do we always need the version here?

Reply:
yes since source maps would get updated on every build so maintaining source maps for each version